src/main/java/com/google/sps/data/Room.java

```diff
@@ -73,6 +73,36 @@ private static Entity toEntity(Room room){
         return newRoom;
     }
 
+    //Creates a room object from a Datastore Key
+    public static Room fromKey(long roomID) {
+        Key roomKey = KeyFactory.createKey("Room", roomID);
+        try {
+            return Room.fromEntity(datastore.get(roomKey));
+        } catch (EntityNotFoundException e) {
+            System.out.println(e.toString());
+        }
+        return null;
+    }
```

Comment:
there is a 'fromRoomId' function that performs this functionality, it also has a more descriptive function name